src/ucp/tag/rndv.c

```diff
@@ -130,11 +132,13 @@ void ucp_tag_send_start_rndv(ucp_request_t *sreq)
     sreq->send.uct.func = ucp_proto_progress_rndv_rts;
 }
 
-static void ucp_rndv_send_ats(ucp_request_t *rndv_req, uintptr_t remote_request)
+void ucp_rndv_send_ats(ucp_request_t *rndv_req, uintptr_t remote_request)
```

Comment:
why static was removed  ? I don't think this change introduces additional external calls.